src/App.css

```diff
@@ -1,22 +1,73 @@
-.App {
+
+.app {
   text-align: center;
+  display: flex;
+  height: 90%;
 }
 
-.App-logo {
-  height: 40vmin;
+.app-header {
+  height: 10%;
+  background-color: #1a1b1b;
+  display: flex;
+  margin-bottom: 5px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.heading {
+  font-size: xx-large;
+  font-style: oblique;
+  font-family: cursive;
+  color: rgb(32, 127, 204);
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  margin: 20px;
+  margin-top: 40px;
```

Comment:
same here. use shorthand margin property to set the top `margin` rather than using `margin-top` property.